### ISSUE TITLE
Lecture questions about time series data

### ISSUE THREAD

**Issue by hmcmonagle** [QERM514/assistance#17]
Hi! I have a question about autocorrelation in time series. As in lecture (screen shot attached), what does it mean to have a lag of 15 and a correlation of -0.5? I could see why, in this example, tree ring growth from one year might be correlated with tree ring growth from the following year. But in reality why might you have a correlation with a 10-15 year time lab? Or does the negative correlation value just mean even less of a correlation, as opposed to a negative correlation?

![image](https://user-images.githubusercontent.com/43573863/79907836-47e40a80-83e8-11ea-9d85-e2a483ad3819.png)

**Comment by mdscheuerell** [comment]
This is a good Q that we deal with a lot in time series analyses (and we cover in detail in our course). To start with the example fig from lec that you showed above, the y-axis is indeed correlation, which lies on the interval [-1,1]. So to answer your last Q first, a value of -0.5 is indeed a correlation of -0.5.

Re: the number of lags at which to examine an ACF, 15 is simply the default for `acf()`. That said, it also depends on the time scale at which we're working. If *t* is measured in years, then it would seem odd to wonder about something like correlated precipitation effects 15 years apart. However, it might make more sense if we're talking about hours.

There is another detail here that I did not mention with respect to interpreting ACFs. It's entirely possible, and actually reasonable, that there is no meaningful correlation between *X* at time *t* and *X* at some time *t+h* later where *h* is large. It helps to think about a deterministic example first. Consider the transitive property of math, which states

if A = B & B = C, then A = C.

Now let's add a bit of error to each, such that

(A + e) ~= (B + e) and (B + e) ~= (C + e), so that (A + e) ~= (C + e).

So, if we created a time series with an autoregressive model wherein the dependency of *X* at time *t* was **solely** a function of *X* at time *t-1* (ie, one time step previous), such that

x_t = phi * x _{t-1} + e_t with e_t ~ N(0, sigma^2),

the ACF would look just like the one above. Go ahead and try this:

```
set.seed(123)
TT <- 50
phi <- 0.7  # 0 < phi < 1
xx <- ee <- rnorm(TT, 0, 1)
for(t in 2:TT) {
  xx[t] <- phi * xx[t-1] + ee[t]
}
acf(xx)
```

Check out the ACF plot you just produced. This is unexpected: we see significant correlation at lags 2 & 3 (and significant negative correlation at lags 9-12) even though our generating model has no such structure; it only specifies a correlation at lag = 1.

Thus, to uncover the real underlying autoregressive structure in the data, we instead turn to a *partial* autocorrelation function (PACF), which removes the dependence induced by intermediate lags.
